src/store/actions/index.js

```diff
@@ -37,3 +37,10 @@ export const deleteBlockAction: BlockActionType = uid => ( {
 	type: ActionTypes.BLOCK.DELETE,
 	uid: uid,
 } );
+
+export const parseBlocksAction: BlockActionType = (payload, parser) => ( {
+	type: ActionTypes.BLOCK.PARSE,
+	uid: '',
```

Comment:
`uid` can be removed, it is never used. When parsing blocks from HTML you can get multiple blocks and as far as I see, it replaces all content.

Reply:
If I remove `uid`, `flow` gets angry saying that `'uid' is missing in object literal`, since `BlockActionType` expects a `uid` property.
That's one reason why I thought that a new reducer operating in the same store might be a good idea.

Instead, I'm inclined to define a `ParseActionType` to keep it simple, remove `uid` and keep `flow` happy.